app/viewModels/P5/arrival/ArrivalNotificationWithFunctionalErrorsP5ViewModel.scala

```diff
@@ -43,13 +43,23 @@ case class ArrivalNotificationWithFunctionalErrorsP5ViewModel(tableRows: Seq[Seq
     )
   }
 
-  def paragraph2Prefix(implicit messages: Messages): String = messages("arrival.ie057.review.notification.message.paragraph2.prefix")
-  def paragraph2Link(implicit messages: Messages): String   = messages("arrival.ie057.review.notification.message.paragraph2.link")
+  def paragraph2(implicit messages: Messages): String = if (multipleErrors) {
+    messages(
+      "arrival.ie057.review.notification.message.paragraph2.plural"
+    )
+  } else {
+    messages(
+      "arrival.ie057.review.notification.message.paragraph2.singular"
+    )
+  }
```

Comment:
Is this logic needed? `arrival.ie057.review.notification.message.paragraph2.plural` and `arrival.ie057.review.notification.message.paragraph2.singular` appear to be exactly the same?

Reply:
Done...